packages/@guardian/source-react-components-development-kitchen/src/toggle-switch/styles.ts

```diff
@@ -17,95 +34,112 @@ export const buttonStyles = (labelPosition: LabelPosition) => css`
- */
+export const toggleStyles = (format?: ArticleFormat): SerializedStyles => {
+	return css`
+		width: 44px;
+		height: 22px;
+		border-radius: 16px;
+		box-sizing: unset;
 
-export const toggleStyles = css`
-	width: 2.75rem;
-	height: 1.5rem;
-	border-radius: 15.5px;
+		/* this will go away when resets have been standardised */
+		&:before,
+		&:after {
+			box-sizing: border-box;
+		}
 
-	/* this will go away when resets have been standardised */
-	&:before,
-	&:after {
-		box-sizing: border-box;
-	}
+		&:before {
+			content: '';
+			position: absolute;
+			top: 5px;
+			height: 11px;
+			width: 6px;
+			right: 10px;
+			opacity: 0;
+			border-bottom: 2px solid ${success[400]};
+			border-right: 2px solid ${success[400]};
+			transform: rotate(45deg);
+			transition-property: opacity;
+			transition-duration: 0.2s;
+		}
 
-	&:before {
-		content: '';
-		position: absolute;
-		top: 6px;
-		height: 11px;
-		width: 6px;
-		right: 10px;
-		opacity: 0;
-		border-bottom: 2px solid ${success[400]};
-		border-right: 2px solid ${success[400]};
-		transform: rotate(45deg);
-		transition: opacity 0.1s ease-in;
-	}
+		&:after {
+			height: 18px;
+			width: 18px;
+			top: 2px;
+			left: 4px;
+		}
 
-	&:after {
-		height: 18px;
-		width: 18px;
-		top: 3px;
-		left: 4px;
-	}
+		&[aria-checked='false'] {
+			background-color: ${format ? toggleBackground : neutral[46]};
+			border: 1px solid ${format ? toggleBorder : neutral[46]};
+		}
 
-	&[aria-checked='false'] {
-		background-color: rgba(153, 153, 153, 0.5);
-	}
+		&[aria-checked='false']:before {
+			transition-delay: 0;
+		}
 
-	&[aria-checked='true'] {
-		background: ${success[500]};
-	}
+		&[aria-checked='true'] {
+			background-color: ${success[400]};
+			border: 1px solid ${format ? toggleBorderGreen : success[400]};
+		}
 
-	&[aria-checked='true']:before {
-		opacity: 1;
-		z-index: 1;
-	}
+		&[aria-checked='true']:before {
+			opacity: 1;
+			z-index: 1;
+			transition-delay: 0.2s;
+		}
 
-	&[aria-checked='true']:after {
-		left: 1.375rem;
-		background: ${neutral[100]};
-	}
-`;
+		&[aria-checked='true']:after {
+			left: 22px;
+			background: ${neutral[100]};
+		}
+	`;
+};
 
-export const labelStyles = css`
-	${textSans.small()};
-	display: flex;
-	color: ${neutral[7]};
+export const labelStyles = (
+	fontSize: ToggleSwitchFontSize,
+	fontWeight: ToggleSwitchFontWeight,
+	format?: ArticleFormat,
+): SerializedStyles => css`
+	${textSans[fontSize]({ fontWeight })};
+	color: ${format ? neutral[100] : neutral[7]};
+	display: inline-flex;
+	justify-content: space-between;
 	align-items: center;
 	cursor: pointer;
 	user-select: none;
 	position: relative;
 `;
 
+export const labelBorderStyles = (
+	format?: ArticleFormat,
+): SerializedStyles => css`
+	border-top: 1px solid ${format ? neutral[100] : neutral[46]};
+	padding-top: ${space[1]}px;
+	width: 100%;
+`;
+
 export const tooltipStyles = css`
 	position: absolute;
-	visibility: hidden;
-	/* display: none; */
+	display: none;
```

Comment:
I think `display: none;` will remove the tooltip from the accessibility tree, so a screen reader couldn't access it. Do we want them to be read out?

Reply:
Good point - this will be superseded in the next changes to add a fallback so not sure how important it is but shouldn't be introducing accessibility regressions, I'll have a look at it.

Reply:
Put it back to using `visibility`

Reply:
nice one — I wonder if `visuallyHidden` might be useful here, or if it doesn't work for this use-case 🤔

Reply:
Sorry maybe wasn't clear enough - this will be replaced in the coming weeks so probably doesn't need changing at this stage.

Reply:
ah, sounds good!  nice one :)